mico-core/src/main/java/io/github/ust/mico/core/DependsOn.java

```diff
@@ -46,6 +45,18 @@ public DependsOn(Service service, String minVersion, String maxVersion) {
         this.maxVersion = maxVersion;
     }
 
+    /**
+     * Set the dependsOn of the service object to null to avoid recursion
+     *
+     * @return Service object with dependsOn set to null
+     */
+    @JsonProperty("serviceDependee")
+    private Service getDependee() {
+        Service dependee = this.serviceDependee;
+        dependee.setDependsOn(null);
```

Comment:
Assume, we need to know, if this returned services has a dependee. How and where do we get this information from?

Reply:
We would need to make a new request GET /services/{shortName}/{version}. This is to break the possible endless recursion between the dependsOn and service objects.
The commented code you previously commented on was originally for breaking the recursion, but it only works for the same kind of objects.